shared/src/main/kotlin/com/egm/stellio/shared/model/AttributeType.kt

```diff
@@ -0,0 +1,8 @@
+package com.egm.stellio.shared.model
+
+enum class AttributeType {
```

Comment:
to be discussed, not sure it should be in a new file. and the name is not specific enough.